recipes/rauth/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "rauth" %}
+{% set version = "0.7.3" %}
+{% set sha256 = "524cdbc1c28560eacfc9a9d40c59525eb8d00fdf07fbad86107ea24411477b0a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  
+build:
```

Comment:
I apologize if I missed this in the initial review, but I think this package is a candidate to be `noarch: python`

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
I'll update it now